x-pack/solutions/security/plugins/security_solution/server/utils/index_name_parser.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Combines an index name with integration namespaces.
```

Comment:
Can you further clarify what this utility does - specifically: how does it differ from the existing one located here: 

https://github.com/elastic/kibana/blob/da25d13a2ac5be97216e202bf5d6bda6ff920b3b/x-pack/solutions/security/plugins/security_solution/common/endpoint/utils/index_name_utilities.ts#L20

Wondering if this existing one could have been changed to maybe support your use case. Maybe an `options` argument with a property called `endingWith: true`... and when this is `true`, it would include the `*` prior to the `-<namespace>`.

Reply:
Extended in 8022c90a01d3af41fe9cef03854ee8b96606a331
Replaced usage in 0507e12d10a3fb8f9505b94c34689b3193c64ecb